index.html

```diff
@@ -12,94 +12,124 @@
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>hours</span></div>
 					</div>
-					<div class="dash_title"><span>hours</span></div>
-				</div>
-				<!-- ## MINUTES ## -->
-				<div class="dash minutes_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## MINUTES ## -->
+					<div class="dash minutes_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>minutes</span></div>
 					</div>
-					<div class="dash_title"><span>minutes</span></div>
-				</div>
-				<!-- ## SECONDS ## -->
-				<div class="dash seconds_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## SECONDS ## -->
+					<div class="dash seconds_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>seconds</span></div>
 					</div>
-					<div class="dash_title"><span>seconds</span></div>
 				</div>
+				<!-- ## COUNTDOWN TIMER END ## -->
+				<a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video">Buy Tickets</a>
 			</div>
-			<!-- ## COUNTDOWN TIMER END ## -->
-
-			<a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video">Buy Tickets</a>
-
 		</div>
-
 	</section>
-	<!-- ## FULLSCREEN SLIDES ## -->
 
+	<!-- ## SPONSOR SECTION ## -->
+	<section class="qcContainer sponsors-section" id="sponsors">
+	  <div class="row middle-md">
+	    <div class="col-xs-12 col-md-6">
+	      <h2>Our Sponsors</h2>
+	    </div>
+	    <div class="col-xs-12 col-md-6 end-md">
+	      <a href="sponsor-prospectus.pdf" target="_blank" class="prospectus-link">Interested in becoming a sponsor?</a>
+	    </div>
+	  </div>
+	  <div class="row sponsor-level">
+	    <div class="col-xs-12">
+	      <h3>Silver Level Sponsors</h3>
+	    </div>
+	    <div class="col-xs-12 col-sm-4">
+	      <a href="http://cdk-global.com"><img src="assets/img/logos/cdk-logo.png" /></a>
+	      <p><a href="http://cdk-global.com">CDK Global</a> is the largest global provider of integrated information technology and digital marketing solutions to the automotive retail industry.</p>
+	    </div>
+	  </div>
+	  <div class="row sponsor-level">
+	    <div class="col-xs-12">
+	      <h3>Supporters</h3>
+	    </div>
+	    <div class="col-xs-6 col-sm-3">
+	      <img src="assets/img/logos/symphonia-logo.png" style="padding-top: 16px;" />
+	    </div>
+	    <div class="col-xs-6 col-sm-3">
+	      <img src="assets/img/logos/puppet-logo.png" />
+	    </div>
```

Comment:
can we get these logos as links to the company websites? the urls are in the story

also, these companies are contributing somewhere between 1/3-1/5 of what the bronze supporters are, so is it possible to make their logos smaller? if you try it and it looks absurd on mobile, just tell me to take a hike ;)